src/appReview/saga.ts

```diff
@@ -0,0 +1,31 @@
+import InAppReview from 'react-native-in-app-review'
+import { call, put, select, spawn, takeLatest } from 'redux-saga/effects'
+import { lastInteractionTimestampSelector } from 'src/appReview/selectors'
+import { setLastInteractionTimestamp } from 'src/appReview/slice'
+import { Actions as SendActions } from 'src/send/actions'
+import { safely } from 'src/utils/safely'
+import { ONE_DAY_IN_MILLIS } from 'src/utils/time'
+
+export function* setAppReview() {
+  // Quick return if the device does not support in app review
+  if (!InAppReview.isAvailable()) return
```

Comment:
do you also want to include an early return if there is no wallet address? so that it is never displayed during onboarding?

Reply:
Fixed in 12baef6524c0438ddbd1a4ec1f1c31c08be785df and tested in f88199429566a04db42eff25c6fdc58f95c8e298!